test/modules_test.sh

```diff
@@ -73,29 +59,21 @@ test_rb_module() {
 
   echo "puts 'Hello world!'" > repl_tmp/repl.rb
```

Comment:
These `echo` statements that put the `Hello world!` programs into the files are what we are the redundancies we can remove now thanks to the boilerplates

Reply:
Additionally, now that all modules have the `Hello world!` boilerplate, we can add the `assertContains` check to all tests that makes sure the files generated by repl contain `Hello world!`

Reply:
hello! Updated code and added the additional check.